handbook/people/README.md

```diff
@@ -176,6 +176,38 @@ We use the Zoom add-on for Google Calendar to schedule Zoom meetings when we cre
 
 We configure our Zoom meetings to let participants join before the host starts the meeting. We do this to make sure meetings start on time, even if the host isn't there.
 
+### Gong
+
+Capturing video from meetings with customers, propsects, and community members outside the company is an important part of building world-class sales and customer success teams, and is a widespread practice across the industry.
+
+At Fleet, we use Gong to capture Zoom meetings and share them company-wide if a team member with a Gong license attends certain meetings, generally those with at least one person from outside of Fleet in attendance.  
+
+While some other Fleeties may have a Gong seat if it becomes necessary in the course of their work, the typical use case will be Fleet employees on the company's sales, customer success, or customer support teams. You will be notified anytime you join a recorded call with an audio message announcing that "this meeting is being recorded" or "recording in progress".  To stop a recording, the person on the call with a Gong seat can press "Stop". If the call has external participants and is being recorded, this call will end up being stored in Gong for future use. 
```

Comment:
@hollidayn We can stick to present tense, rather than future, here and throughout, since everything is in draft and this is live today.